openc3-cmd-tlm-api/app/controllers/scopes_controller.rb

```diff
@@ -23,4 +23,19 @@ class ScopesController < ModelController
   def initialize
     @model_class = OpenC3::ScopeModel
   end
+
+  def create(update_model = false)
+    return unless authorization('superadmin')
+    super(update_model)
+  end
+
+  def destroy
+    return unless authorization('superadmin')
+    begin
+      result = OpenC3::ProcessManager.instance.spawn(["ruby", "/openc3/bin/openc3cli", "destroyscope", params[:id]], "scope_uninstall", params[:id], Time.now + 2.hours, scope: 'DEFAULT')
```

Comment:
Seems weird to have to give a scope: 'DEFAULT' to the process to destroy scopes

Reply:
The DEFAULT scope is also going to be the kindof "global scope" that owns Scopes and Roles.